driver-scala/src/it/scala/org/mongodb/scala/TestMongoClientHelper.scala

```diff
@@ -51,4 +51,12 @@ object TestMongoClientHelper {
   def hasSingleHost: Boolean = {
     TestMongoClientHelper.connectionString.getHosts.size() == 1
   }
+
+  Runtime.getRuntime.addShutdownHook(new ShutdownHook())
+
+  private[mongodb] class ShutdownHook extends Thread {
+    override def run() {
+      TestMongoClientHelper.mongoClient.close()
```

Comment:
@rozza did you intentionally remove the drop of the database?  Not sure it matters, but want to make sure it is intentional.

Reply:
Yeah because its already dropped in the `afterAll` method of the spec test

Reply:
Yeah, I think the reason we have both is in case you Ctrl-C the tests in the middle of the run.

Reply:
I think in that scenario its fine to leave the database in a undefined state